lib/tasks/db.rake

```diff
@@ -14,4 +14,13 @@ namespace :db do
       puts 'Error: Unable to remove Asset#organisation_slug'
     end
   end
+
+  desc "Set the size field for all uploaded assets from the etag"
+  task set_size_for_all_uploaded_assets: :environment do
+    scope = Asset.unscoped.where(state: 'uploaded').where(size: nil)
```

Comment:
Having just approved this PR, I realise I have a question. Why do we only consider `uploaded` assets here? Should we update the size of _all_ assets?

Reply:
@chrisroos I'd added that because any assets that are in the process of being uploaded when this task runs will have their size set by the changes added earlier in the PR. But it's probably an unnecessary optimisation - I'll remove it.